src/scss/partials/_button.scss

```diff
@@ -0,0 +1,28 @@
+/**
+* Header button Styling
+ * =================================
+ * Styles relating to button (like mobile filter button) within header
+ */
+
+ .c-headerButton {
+    border: 0;
+    outline: 0;
+    padding: 0;
+    font-size: 0;
```

Comment:
Should this element be added to the header template as well? 

Also, are some of these styles here to hide text from the user?